### ISSUE TITLE
Performance on linux/osx

### ISSUE THREAD

**Issue by ChrisPritchard** [ChrisPritchard/FSH#14]
Performance of printing is atrocious on unix systems.  Suspect this due to known problems using cursor top.

**Comment by ChrisPritchard** [comment]
[This](https://github.com/dotnet/corefx/blob/master/src/System.Console/src/System/ConsolePal.Unix.cs) details the performance issues on unix systems. Unsure how to get around this, aside from maybe building from scratch a unix terminal writer using a different approach than system.console...

**Comment by ChrisPritchard** [comment]
Its possible this will be resolved with the main release of core 3.0. Ill upgrade then and see whats what.

**Comment by voronoipotato** [comment]
Reducing the amount of writes with a stringbuilder can help if its starting to chug. You can basically buffer them like how RX buffer works.

**Comment by ChrisPritchard** [comment]
Yeah I need to take another look, when I have time. Especially since I now have a plethora of non-windows devices to test on.
